README.md

```diff
@@ -116,6 +116,7 @@ itself) to look for plugins.
 | kube-capacity                 | [looztra/asdf-kube-capacity](https://github.com/looztra/asdf-kube-capacity)                       | [![Build Status](https://travis-ci.org/looztra/asdf-kube-capacity.svg?branch=master)](https://travis-ci.org/looztra/asdf-kube-capacity)                                                |
 | Kubectl                       | [Banno/asdf-kubectl](https://github.com/Banno/asdf-kubectl)                                       | [![Build Status](https://travis-ci.org/Banno/asdf-kubectl.svg?branch=master)](https://travis-ci.org/Banno/asdf-kubectl)                                                                |
 | kubectl-bindrole              | [looztra/asdf-kubectl-bindrole](https://github.com/looztra/asdf-kubectl-bindrole)                 | [![Build Status](https://travis-ci.org/looztra/asdf-kubectl-bindrole.svg?branch=master)](https://travis-ci.org/looztra/asdf-kubectl-bindrole)                                          |
+| kubectx                       | [wt0f/asdf-kubectx](https://gitlab.com/wt0f/asdf-kubectx.git)                                     | [![pipeline status](https://gitlab.com/wt0f/asdf-kubectx/badges/master/pipeline.svg)](https://gitlab.com/wt0f/asdf-kubectx/-/commits/master)                                           |
```

Comment:
The `.git` is not required here.
`[wt0f/asdf-kubectx](https://gitlab.com/wt0f/asdf-kubectx)`